packages/mdc-segmented-button/segment/_segment-theme.scss

```diff
@@ -0,0 +1,148 @@
+//
+// Copyright 2020 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+// Theme: Variables, functions, and mixins required to theme component segment
+
+@use "@material/feature-targeting";
+@use "@material/theme/color" as theme-color;
+@use "@material/theme/custom-properties";
+
+$on-surface: theme-color.$on-surface;
+$surface: theme-color.$surface;
+$primary: theme-color.$primary;
+
+$unselected-ink-color: rgba($on-surface, 0.6);
+$unselected-outline-color: rgba($on-surface, 0.12);
+$unselected-container-fill-color: rgba($surface, 1);
+$selected-ink-color: rgba($primary, 1);
+$selected-outline-color: $unselected-outline-color;
+$selected-container-fill-color: rgba($primary, 0.08);
+
+$height: 36px;
+$min-width: 48px;
+$horizontal-padding: 12px;
+$border-width: 1px;
+$border-radius: 4px;
+$icon-width: 24px;
+$icon-font-size: 18px;
+$label-padding: 6px;
+
+/**
+  * Sets the border color of the segment
+  */
+@mixin outline-color($color, $query: feature-targeting.all()) {
```

Comment:
SassDoc should use Sass comments (`/// foo bar`), using CSS comment (`/** foo bar */`) emits in final output.

See https://github.com/material-components/material-components-web/blob/master/packages/mdc-data-table/_data-table-theme.scss for examples. See [SassDoc](http://sassdoc.com/annotations/) for documentation on Sass annontation.